src/cli/kfc.ts

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { execSync } from "child_process";
+import prompt from "prompts";
+
+import { RootCmd } from "./root";
+
+export default function (program: RootCmd) {
+  program
+    .command("kfc crd [source] [dest]")
```

Comment:
```suggestion
    .command("kfc crd <source> <dest>")
```

Reply:
```
You can specify an option which may be used as a boolean option but may optionally take an option-argument (declared with square brackets like --optional [value]).
```

```
The two most used option types are a boolean option, and an option which takes its value from the following argument (declared with angle brackets like --expect <value>). Both are undefined unless specified on command line.
```

We want to ensure this these values are not optional, [source](https://github.com/tj/commander.js?)